perllib/FixMyStreet/Cobrand/FixMyStreet.pm

```diff
@@ -42,8 +43,21 @@ sub munge_category_list {
 
     # No TfL Traffic Lights category in Hounslow
     my %bodies = map { $_->body->name => 1 } @$contacts;
-    return unless $bodies{'Hounslow Borough Council'};
-    @$options = grep { ($_->{category} || $_->category) !~ /^Traffic lights$/i } @$options;
+    if ( $bodies{'Hounslow Borough Council'} ) {
+        @$options = grep { ($_->{category} || $_->category) !~ /^Traffic lights$/i } @$options;
+    }
+
+    if ( $bodies{'Isle of Wight Council'} ) {
+        my $user = $self->{c}->user;
+        my $b = $self->{c}->model('DB::Body')->for_areas( COUNCIL_ID_ISLEOFWIGHT )->first;
```

Comment:
If you change the %bodies hash above, you can already have the body here, it's already been looked up.